params.R

```diff
@@ -29,6 +31,7 @@ lfpse_categorical <- expr(A001 == '4')
 
 # steis categorical filters (wrap in expr() or set to 0)
 steis_categorical <- expr(type_of_incident == 'Medication incident meeting SI criteria')
+steis_url <- ' ' # Added in URL for steis extract
```

Comment:
changed wording slightly
```suggestion
steis_url <- ' ' # Please add the URL of the steis extract
```